README.md

```diff
@@ -1,24 +1,152 @@
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Budd-Get <a name="about-project"></a>
 
-* Database creation
+**Budd-Get** is a ruby on rails app that helps you track your expenses organized by categories.
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+- `Ruby on Rails`
 
-* Services (job queues, cache servers, search engines, etc.)
+<!-- Features -->
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- Show the list of categories added
+- Add new categories
+- Show the list of transactions by category
+- Add new transactions
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budd-get.onrender.com/)
+- [Video Demostration](https://www.loom.com/share/335925cb77b7422ab1ab105e5ea88403)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started
+To clone the repository in local environment try following steps.
+
+### Prerequisites
+
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Node.js and npm for package installation.
+```
+$ ruby --version && gem --version
+```
+### Setup
+
+Clone the repository using [this link](https://github.com/Gdiazdiaz/My-Budget.git).
+
+### Install
+
+- Ruby
+- Ruby on Rails
+- Postgresql
+
+### Get into development
+
+In the terminal, run these commands to get into development.
+```
+$ bundle install (install dependencies)
+```
+### Usage
+
+To run the project run the command:
+```
+  rails server
+```
+
+<!-- ### Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Glenda Diaz**
+
+- GitHub: [@Gdiazdiaz](https://github.com/Gdiazdiaz)
+- LinkedIn: [Glenda Diaz](www.linkedin.com/in/glendadiazz)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Hamburguer menu**
+- [ ] **Transaction details**
+- [ ] **Profile page**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome! Add suggestions by opening new issues.
+
+Feel free to check the [issues page](../../issues/)..
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Authors would like to thank:
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Coding Partners
```

Comment:
Your  "Acknowledgements" section is missing a credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) 

- [ ] Kindly update the acknowledgements as it is required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).